api/swagger.yaml

```diff
@@ -4760,6 +4760,56 @@ paths:
           type: "string"
           required: true
       tags: ["Image"]
+  /images/{name}/manifest:
+    get:
+      summary: "Get image manifest"
+      description: "Return image digest and manifest information by contacting the registry."
+      operationId: "ImageManifest"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "digest and manifest json"
+          schema:
+            type: "object"
+            x-go-name: ManifestInspect
+            required: [Digest, ManifestMediaType, Manifest, ConfigJSON]
+            properties:
+              Digest:
+                type: "string"
+                x-nullable: false
+              ManifestMediaType:
+                type: "string"
+                x-nullable: false
+              Manifest:
+                type: "string"
+                x-nullable: false
+              ConfigJSON:
+                type: "string"
+          examples:
+            application/json:
+              Digest: "sha256:58e1a1bb75db1b5a24a462dd5e2915277ea06438c3f105138f97eb53149673c4"
+              ManifestMediaType: "application/vnd.docker.distribution.manifest.v2+json"
+              Manifest:
+              ConfigJSON:
```

Comment:
@nishanttotla could you use put a sample one ? inlined like `'{"key":"value","key2":"value2"}'`